pypowsybl/network.py

```diff
@@ -4124,6 +4124,39 @@ def remove_extensions(self, extension_name: str, ids: _Union[str, _List[str]]) -
             ids = [ids]
         _pp.remove_extensions(self._handle, extension_name, ids)
 
+    def get_connectables_order_positions(self, voltage_level_id: str) -> _DataFrame:
```

Comment:
**A general comment about the feature**
there is actually already a generic mechanism to expose extensions data, through the `network.get_extensions` method.
However, it's indeed not very user-friendly because you need to know the name of the extension and so on.
So exposing their data through more targetted user-friendly methods makes sense.

But still, we should probably ALSO expose the `ConnectablePosition` extension through the generic mechanism, to be able to view the directions, the feeder name, and to be able to change them.
This should be done in another branch.

**About the place of the method**
I wonder if we should leave it in the Network class, because it's very, very, specific.

Reply:
Tracked it in #495